.github/workflows/changelog-and-release.yml

```diff
@@ -16,19 +16,19 @@ on:
       version_type:
         description: 'Create a ''minor'' or ''micro'' release?'
         required: true
-        default: 'micro'
+        default: 'minor'
```

Comment:
please dont change this, or i can restore it later
several times it happened that you published a minor version instead of micro

Reply:
As you wish. :-) Although differences to the standard makes automated updates of this scripts more complicated.